editor/src/views/Crashes/SwapAddressButton.js

```diff
@@ -0,0 +1,65 @@
+import React, { useState } from "react";
+import { Button } from "reactstrap";
+import ConfirmModal from "../../Components/ConfirmModal";
+import { UPDATE_CRASH } from "../../queries/crashes";
+
+const SwapAddressButton = ({ data, crashRefetch, ...props }) => {
```

Comment:
it'd be nice to tighten this up a tiny bit, and pass `crash` directly into this component and add `client` so that we don't need `...props`.